spectrochempy/analysis/mcrals.py

```diff
@@ -500,34 +493,44 @@ def __init__(
     ):
         if len(args) > 0:
             raise ValueError(
-                "Passing arguments such as MCRALS(X , profile) "
+                "Passing arguments such as MCRALS(X, profile) "
                 "is now deprecated. "
-                "Instead, use MCRAL() followed by MCRALS.fit(X , profile). "
+                "Instead, use MCRAL() followed by MCRALS.fit(X, profile). "
                 "See the documentation and examples"
             )
 
         # warn about deprecation
         # ----------------------
         # We use pop to remove the deprecated argument before processing the rest
-        # TODO: These arguments should be removed in version 0.6 probably
+        # TODO: These arguments should be removed in version 0.7 probably
 
         # verbose
         if "verbose" in kwargs:
-            deprecated("verbose", replace="log_level='INFO'", removed="0.6.5")
+            deprecated("verbose", replace="log_level='INFO'", removed="0.7")
```

Comment:
if you agree, we remove deprecated elements only when we do  major release (0.7, ...)

Reply:
agreed !